easybuild/easyconfigs/b/Boost/Boost-1.70.0-CrayGNU-19.10-python3.eb

```diff
@@ -46,7 +41,7 @@ patches = ['Boost-1.70.0-python%s.patch' % local_py_maj_ver]
 # for python3 the corresponding lib is libboost_python36.so
 sanity_check_paths = {
     'files': ['lib/libboost_system.so', 'lib/libboost_mpi.so',
-              'lib/libboost_python%s%s.so' % (local_py_maj_ver, local_py_min_ver)],
+              'lib/libboost_python%%(pymajver)s%s.so' % _pyminver],
```

Comment:
Thanks @ekouts: same comment here as above for `majver` and `minver`, just to have the same formatting, if possible